pkg/doc/bbs/bbs12381g2pub/internal/bls12_381.go

```diff
@@ -0,0 +1,26 @@
+/*
+Taken from https://github.com/kilic/bls12-381
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
```

Comment:
Please add comments on what commit this file is from and what changes have been made to this file.

Reply:
I changed this to
```
/*
Taken from https://github.com/kilic/bls12-381/blob/master/arithmetic_fallback.go
(rev a288617c07f1bd60613c43dbde211b4a911e4791)

SPDX-License-Identifier: Apache-2.0
(https://github.com/kilic/bls12-381/blob/master/LICENSE)
*/
```